src/docs-style-guide.rst

```diff
@@ -363,6 +513,28 @@ Therefore, avoid the use of personal pronouns whenever possible. They are often
   - open the app
   - complete the survey
 
+.. code-block:: python
+  :class: style-checks
+
+  @memoize
+  def check_pronoun(text):
+      """Avoid using third-person personal pronouns."""
+      err = "style-guide.personal-pronoun"
+      msg = "Avoid using third-person personal pronouns like \"he\", \"she\". "
```

Comment:
As a point of Python coding style, this concatenation thing is weird way to write a string, rather than just assigning it all at once.